tools/myst/enc/enc.c

```diff
@@ -394,6 +451,17 @@ static long _enter(void* arg_)
                 assert(0);
             }
             have_config = true;
+            // oe_host_printf("enclave config=%s size=%ld\n", (char *)r.data,
+            // strlen((char *)r.data));
+            myst_sha256(&config_sha256, r.data, strlen((char*)r.data));
```

Comment:
Per discussion, the hashing target should be something passed in instead of data inside the enclave memory. The config here seems resides in the enclave memory (a region). Can you double check?